joltify/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "joltify",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.joltify.io/",
+  "pretty_name": "Joltify",
+  "chain_id": "joltify_1729-1",
+  "bech32_prefix": "jolt",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Done registration.